lib/Target/DebugInfo/EmitHGLDD.cpp

```diff
@@ -160,12 +160,41 @@ static bool makePathRelative(StringRef path, StringRef relativeTo,
   return outputIt == outputEnd;
 }
 
+static StringRef legalizeName(StringRef name, llvm::StringMap<size_t> &ns) {
+  return sv::legalizeName(name, ns, true);
```

Comment:
nit: Can you add a comment why it uses sv::legalizeName?

Reply:
Great point, thanks. I have added a comment.